stripe/build.gradle

```diff
@@ -40,8 +40,8 @@ dependencies {
     testImplementation 'androidx.test:core:1.2.0'
     testImplementation 'org.json:json:20190722'
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
-    testImplementation "org.jetbrains.kotlin:kotlin-test-junit:1.3.50"
-    testImplementation "org.jetbrains.kotlin:kotlin-test-annotations-common:1.3.50"
+    testImplementation 'org.jetbrains.kotlin:kotlin-test-junit:1.3.60'
```

Comment:
would this ever not be the same as kotlinVersion ?

Reply:
Shouldn't be, I'll update in followup